src/index.html

```diff
@@ -3,11 +3,39 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product" data-qa="card">
+      <div class="product__photo"></div>
+      <h1 class="product__title"> APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h1>
```

Comment:
```suggestion
      <h2 class="product__title">
        APPLE A1419 iMac 27" Retina <br>
        5K Monoblock (MNED2UA/A)
      </h2>
```

Reply:
isn't fixed. change tag

Reply:
not fixed

Reply:
yes, you havent now but in nex task you will be use this card as a component and there it will be looks wrong, and by the way it not logical when card's name its head title of all pages